build.gradle

```diff
@@ -23,9 +23,8 @@ plugins {
 }
 
 ext.ballerinaLangVersion = project.ballerinaLangVersion
-ext.puppycrawlCheckstyleVersion = "8.18"
-ext.testngVersion = "6.14.3"
-ext.slf4jVersion = "1.7.30"
+ext.puppycrawlCheckstyleVersion = project.puppycrawlCheckstyleVersion
+ext.checkstyleToolVersion = project.checkstyleToolVersion
 ext.stdlibRegexVersion = project.stdlibRegexVersion
 ext.stdlibOsVersion = project.stdlibOsVersion
 ext.stdlibIoVersion = project.stdlibIoVersion
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-reflect/pull/47/commits/8f05ed77dca94e9d0268634e27c526e97151ece4